### DISCUSSION TITLE
请求战地2100

### DISCUSSION THREAD

**Discussion by yahvk-cuna** [Gamuxorg/bbs#360] [投递和请求游戏]
在 ArchLinux 上有字体问题，不知道用 AppImage 能不能解决

**Comment by shouhuanxiaoji** [comment]
晚上可以试一试

**Comment by shouhuanxiaoji** [comment]
添加到了站上，试一下

**Comment by yahvk-cuna** [comment]
/tmp/.mount_warzonyaKZxh/usr/bin/warzone2100: error while loading shared libraries: libjson-c.so.2: cannot open shared object file: No such file or directory

我系统上的是 libjson-c.so.3

**Comment by yahvk-cuna** [comment]
降级 json-c 之后，看起来没有其他问题

**Comment by shouhuanxiaoji** [comment]
字体也没问题？

**Comment by yahvk-cuna** [comment]
没问题

**Comment by yahvk-cuna** [comment]
希望能解决一下 libjson-c 的问题

**Comment by shouhuanxiaoji** [comment]
似乎是没有捕捉到缺失的这个依赖，我今晚重新打包一下。你再试试，可以来QQ群274328087一起参与

**Comment by shouhuanxiaoji** [comment]
重新打包了一下，试一下

**Comment by yahvk-cuna** [comment]
还是不行

2018-02-08 20:45 GMT+08:00 小鸡 <notifications@github.com>:

**Comment by shouhuanxiaoji** [comment]
我已经把这个打包进游戏里了。如果还是不行，大概是更底层库的问题

**Comment by yahvk-cuna** [comment]
我解包发现里面的 `libjson-c.so.2` 是链接到  `/home/deepin/Desktop/warzone2100/warzone2100.AppDir/lib/x86_64-linux-gnu/libjson-c.so.2.0.0`

- libjson-c.so.2
- libncursesw.so.5
- libpcre.so.3
- libpng12.so.0
- libslang.so.2
- libtinfo.so.5
- libwrap.so.0

都是这样

还有一个不应该存在的 `libpng12(副本).so.0`

**Comment by shouhuanxiaoji** [comment]
是链接乱了。。嗯 可以修复一下





在 2018-02-09 19:44:43，"DAStudio.71e6fd52" <notifications@github.com> 写道：


我解包发现里面的 libjson-c.so.2 是链接到 /home/deepin/Desktop/warzone2100/warzone2100.AppDir/lib/x86_64-linux-gnu/libjson-c.so.2.0.0

libjson-c.so.2
libncursesw.so.5
libpcre.so.3
libpng12.so.0
libslang.so.2
libtinfo.so.5
libwrap.so.0

都是这样

还有一个不应该存在的 libpng12(副本).so.0
